product_pricelist_revision/models/pricelist.py

```diff
@@ -24,15 +25,11 @@ class ProductPricelistItem(models.Model):
         string="Variation %",
     )
 
-    @api.model
-    def _search_name(self, operator, value):
-        return [
-            "|",
-            "|",
-            ("categ_id", operator, value),
-            ("product_tmpl_id", operator, value),
-            ("product_id", operator, value),
-        ]
+    def _auto_init(self):
+        other_fields = ["categ_id", "product_tmpl_id", "product_id"]
+        for item in other_fields:
+            self.env["product.pricelist.item"]._rec_names_search.append(item)
```

Comment:
Hi,
It was not possible to do something like that?
 ```suggestion
         self.env["product.pricelist.item"]._rec_names_search.extend(other_fields)
```

Reply:
Yes, it's better indeed, thanks